hudi-common/src/test/java/org/apache/hudi/common/table/string/TestHoodieActiveTimeline.java

```diff
@@ -160,6 +159,8 @@ public void testTimelineOperations() {
         .filterCompletedInstants().findInstantsInRange("04", "11").getInstants().map(HoodieInstant::getTimestamp));
     HoodieTestUtils.assertStreamEquals("", Stream.of("09", "11"), timeline.getCommitTimeline().filterCompletedInstants()
         .findInstantsAfter("07", 2).getInstants().map(HoodieInstant::getTimestamp));
+    HoodieTestUtils.assertStreamEquals("", Stream.of("01", "03", "05"), timeline.getCommitTimeline().filterCompletedInstants()
```

Comment:
nit: Add non-empty message for the assertion.

Reply:
fixed